### ISSUE TITLE
Bazel build is missing .coveragerc

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#437]
Bazel builds seem to not have a [.coveragerc](https://github.com/googleapis/gapic-generator-python/tree/master/gapic/templates). This file has been around for a while so I don't think it's a version issue.

```
bazel build //google/cloud/documentai/v1beta2:documentai-v1beta2-py
```

Contents of tar (just missing `.coveragerc`)
```
$ ls -a
.  ..  docs  google  MANIFEST.in  mypy.ini  noxfile.py  scripts  setup.py  tests
```

Resolved in #550: ~~Unit test subdirs are also missing the `__init__.py`. https://github.com/googleapis/gapic-generator-python/blob/master/gapic/templates/tests/unit/gapic/%25name_%25version/%25sub/__init__.py~~

CC @vam-google

**Comment by arithmetic1728** [comment]
any updates as we are out of slo

**Comment by busunkim96** [comment]
@software-dov @vam-google Friendly ping. We now use Bazel in almost all the Python repositories so this is more of a problem.

When I last tested this the `.coveragerc` was present when I ran the generator locally or via the Docker image. It only seems to disappear when `bazel` was used.

Maybe the `.coveragerc` is dropped somewhere here? 
https://github.com/googleapis/gapic-generator-python/blob/f28b43da949bbfb904530c794187c840f0b90a0c/rules_python_gapic/py_gapic_pkg.bzl#L26-L36

**Comment by hkdevandla** [comment]
@vam-google @software-dov , any updates on this? We are out of SLO on this. Also, it seems important to fix this as we started to build libraries using bazel. thanks!

**Comment by hkdevandla** [comment]
@vam-google, any updates on this?

**Comment by vam-google** [comment]
@hkdevandla Working on it

**Comment by vam-google** [comment]
Apparently the issue was in bazel's `glob()` funciton not including files starting with dot (`.`) by default, probably assuming those are  supposed to be hidden files.

**Comment by vam-google** [comment]
Fixed in #723. Closing.
